src/components/structures/MessagePanel.tsx

```diff
@@ -52,8 +64,8 @@ function shouldFormContinuation(prevEvent, mxEvent) {
 
     // Some events should appear as continuations from previous events of different types.
     if (mxEvent.getType() !== prevEvent.getType() &&
-        (!continuedTypes.includes(mxEvent.getType()) ||
-            !continuedTypes.includes(prevEvent.getType()))) return false;
+        (!continuedTypes.includes(mxEvent.getType() as EventType) ||
```

Comment:
shouldn't it be returning an EventType already?

Reply:
Returns `EventType | string` given the EventType enum is only the specced subset